public/js/styleAdmin.js

```diff
@@ -0,0 +1,11 @@
+// $('#btn-logout').click(function(){
```

Comment:
[rank C] css & js file name should be follow underscore style.
and please remove comment code if you don't need to use it anymore.